packages/react/src/primitives/Icon/Icon.tsx

```diff
@@ -35,7 +35,7 @@ const IconPrimitive: Primitive<IconProps, 'svg'> = (
     _children = children;
   }
   if (paths) {
-    _children = paths.map((path) => <path {...path} />);
+    _children = paths.map((path, index) => <path {...path} key={index} />);
```

Comment:
@zchenwei Did we decide on a better approach than using the `index` as the `key`?

Reply:
We have a util called `useStableId` that could be used to generate id for `key` prop. However, that util function is dependent on Radix `useId`, which we want to remove it in favor our own implementation. I am not sure how much we progress there.